ORBIT/phases/design/electrical_export.py

```diff
@@ -492,32 +567,87 @@ def calc_topside_mass_and_cost(self):
 
         Parameters
         ----------
-        topside_fab_cost_rate : int | float
         topside_design_cost: int | float
         """
 
         _design = self.config.get("substation_design", {})
-        topside_fab_cost_rate = _design.get("topside_fab_cost_rate", 14500)
-        topside_design_cost = _design.get("topside_design_cost", 4.5e6)
+        # topside_fab_cost_rate = _design.get("topside_fab_cost_rate", 14500)
+        # topside_design_cost = _design.get("topside_design_cost", 4.5e6)
```

Comment:
Is this being preserved, or was it just missed in the cleanup?

Reply:
I guess it was preserved, but it probably doesn't need to be preserved this we use this in `oss_design.py`

Reply:
I'm just checking, though by the time it gets to the main branch we might want to clean it up. This is definitely your call, I just wanted to understand the rationale.